### PR TITLE
[14.0][MIG] sale_order_global_stock_route: Migration to 14.0

### PR DESCRIPTION
**Pull request by Abranes**
Standard migration to 14.0

### CODE REVIEW

File: sale_order_global_stock_route/README.rst [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+For use this module you have to activate the option Settings > Inventory
```
Comment (rousseldenis) [inline sale_order_global_stock_route/README.rst]:
```suggestion
To use this module you have to activate the option Settings > Inventory
```

File: sale_order_global_stock_route/models/sale_order.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    def product_id_change(self):
+        res = super().product_id_change()
+        self.route_id = self.order_id.route_id
+        return res
+
+    @api.model_create_multi
+    def create(self, vals_list):
+        for vals in vals_list:
+            if not vals.get("route_id", False):
+                order = self.env["sale.order"].browse(vals["order_id"])
```
Comment (rousseldenis) [inline sale_order_global_stock_route/models/sale_order.py:50]: I would avoid filling in value entry if route_id is not defined on sale order.

File: sale_order_global_stock_route/models/sale_order.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        return res
+
+
+class SaleOrderLine(models.Model):
+    _inherit = "sale.order.line"
+
+    @api.onchange("product_id")
+    def product_id_change(self):
+        res = super().product_id_change()
+        self.route_id = self.order_id.route_id
```
Comment (rousseldenis) [inline sale_order_global_stock_route/models/sale_order.py]: I would have added a condition if route_id is defined on sale order before modifying.

**Review by rousseldenis — CHANGES_REQUESTED** [review]
@Abranes Thanks for this

Code review.

File: sale_order_global_stock_route/README.rst (hunk 1 above)
Comment (Abranes) [reply sale_order_global_stock_route/README.rst]: Done!

File: sale_order_global_stock_route/models/sale_order.py (hunk 3 above)
Comment (Abranes) [reply sale_order_global_stock_route/models/sale_order.py]: Done!

File: sale_order_global_stock_route/models/sale_order.py (hunk 2 above)
Comment (Abranes) [reply sale_order_global_stock_route/models/sale_order.py:50]: Done!

**Comment by Abranes** [comment]
Changes attended @rousseldenis

**Review by rousseldenis — APPROVED** [review]

**Review by hildickethan — APPROVED** [review]
LGTM

**Pull request merged**